src/Apps/Artist/Routes/Overview/Components/__tests__/ArtistCareerHighlights.jest.tsx

```diff
@@ -80,6 +79,8 @@ describe("ArtistCareerHighlights", () => {
       expect.stringContaining("partner/number-one-best-gallery")
     )
     expect(screen.getByText("this artist rocks")).toBeInTheDocument()
-    expect(screen.queryByText(CV_LINK_TEXT)).toBeInTheDocument()
+    expect(
+      screen.queryByText("See all past shows and fair booths")
+    ).toBeInTheDocument()
```

Comment:
I guess we have to add an `import "System/i18n/i18n"` for this to work here
(I'm actually wondering if maybe it would be better to import this on `setupTestWrapper`? not sure 🤔 )

Reply:
We could import it lower down, in `src/tests.ts`!

Reply:
Good call, I just updated this individual test with that import and it went green. Then I took it out and moved it to the test setup file as @damassi mentioned and that worked too - thanks all!

Reply:
oh hrm, I now see that adding this to the test.ts file breaks a few tests:

```
src/Apps/Search/Components/__tests__/SearchMeta.jest.tsx
src/Apps/Artist/Components/__tests__/ArtistMeta.jest.tsx
src/Apps/Consign/Routes/SubmissionFlow/ContactInformation/__tests__/ContactInformation.jest.tsx
src/lib/__tests__/openAuthModal.jest.ts
```

What do yall think? Maybe I can separate the test setup improvement from this larger ticket?

Reply:
Yeah, I agree with the idea of handling the test setup improvement in a separate PR

Reply:
@damassi @lidimayra ok cool then I'll reverse out that test setup change and get this landed. Can come back through with the improvement later. 👍